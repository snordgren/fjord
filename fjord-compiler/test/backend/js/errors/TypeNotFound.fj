module TypeNotFound

myUnknownValue : UnknownTypeName
myUnknownValue = 5
