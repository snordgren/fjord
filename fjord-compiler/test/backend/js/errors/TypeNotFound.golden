TypeNotFound.fj:3:18:
  |
3 | myUnknownValue : UnknownTypeName
  |                  ^^^^^^^^^^^^^^^^
type not found UnknownTypeName
